src/BitBucketOnPrem.ts

```diff
@@ -30,52 +35,102 @@ export interface BitBucketOnPremInput {
 const fetchNoCache = (requestInfo: RequestInfo, requestInit: RequestInit) => {
     requestInit = requestInit || {};
     if (!requestInit?.cache) {
-        requestInit.cache = 'no-store'
+        requestInit.cache = 'no-store';
     }
     if (!requestInit?.credentials) {
         requestInit.credentials = 'omit';
     }
-    return fetch(requestInfo.toString().replace('scm/', '').replace('scm%2F',''), requestInit);
-}
+    return fetch(requestInfo.toString().replace('scm/', '').replace('scm%2F', ''), requestInit);
+};
+
+export const getFileTreeByPath =
+    async ({url, accessToken, projectKey, repoName, commit, filePath}: BitbucketOnPrem): Promise<string[]> => {
+        const templateUrl: string = addSlugToUrl('rest/api/1.0/projects/:projectKey/repos/:repoName/browse', filePath);
+        // build url without filePath (Url slug needs to be "src/folder" but UrlAssembler will turn it into unicode)
```

Comment:
Wait, did not see this comment. Can you check if it works? It still needs to happen before UrlAssembler imo but maybe there is a way to tell it not to parse it to unicode.

Reply:
comment is no longer needed since we add the slug to the template before going through the url assembler